src/Forms/Select/index.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../shared';
+
+.select {
+  position: relative;
+
+  &::after {
+    color: $color-dark;
+    content: '\221F';
+    display: inline-block;
+    font-size: $font-size-xs;
+    height: 0;
+    pointer-events: none;
+    position: absolute;
+    right: $spacing-xxl;
```

Comment:
I remember that this positioning/sizing etc. breaks once the responsive font sizes kick it. Might already be fixed in the brand-styles master branch.